Directory.Build.props

```diff
@@ -14,6 +14,9 @@
     <AnalysisMode>Recommended</AnalysisMode>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+    <BeforeTargetFrameworkInferenceTargets>$(RepositoryEngineeringDir)BeforeTargetFrameworkInference.targets</BeforeTargetFrameworkInferenceTargets>
+    <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
```

Comment:
You shouldn't need to set this. This property is only needed to tell the SDK that you are referencing a newer .NETCoreApp version that what's defined as the maximum supported on by the SDK.

Reply:
This was one of the errors I mentioned in the PR description. If I don't define it, then restore fails saying that NETCoreAppMaximumVersion is set to a non-numeric value of `"."`.

Reply:
I also removed this and it's also working. What!